core/src/elements/mine.rs

```diff
@@ -12,11 +13,11 @@ pub fn install(session: &mut GameSession) {
 #[ulid = "01GPRSBWQ3X0QJC37BDDQXDNF3"]
 pub struct IdleMine;
 
-#[derive(Clone, TypeUlid, Debug, Copy)]
+#[derive(Clone, TypeUlid, Debug)]
 #[ulid = "01GPRSBWQ3X0QJC37BDQXDNASF"]
 pub struct ThrownMine {
-    // How long the mine has been thrown.
-    age: f32,
+    // When this timer reaches the fuse duration, the mine will explode.
```

Comment:
The mine actually doesn't explode when this timer finishes, unless it is colliding with a player.

A more accurate description would be something like: “The mine won't explode until this timer finishes.”

Reply:
Ok I made the suggested updates. Thanks for the feedback! Lmk if there's other changes you'd like me to make. 

Btw is working as a software engineer kind of like this? Like working on tickets/issues and resolving them? 

Are the issues generally _always_ harder than `good first issue` tickets?

Reply:
> Btw is working as a software engineer kind of like this? Like working on tickets/issues and resolving them?

Yeah, I think so, though my only software development experience is on GitHub. 😄

> Are the issues generally always harder than good first issue tickets?

Maybe not always, and it depends a lot on how much experience you have with a certain topic.

This kind of stuff gets a lot easier as you continue to do it. Once you've worked in a certain field/language/project long enough you build familiarity with it, and it doesn't take as many tries before you get it right.

I've learned amost everything I know just by trying to make stuff work, and most of the time I didn't know what I was doing when I started. Now I've done that so many times on so many different projects, that a lot of things aren't so hard.

It's cool because the more you practice and work on different things, the easier it gets to do new things you've never done before.

The most imporant thing is to learn how to learn more, then eventually you can do anything.

Reply:
awesome, thanks for answering my questions.